e2e/search-bar-optimized.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+const getSearchInput = async ({
+  page,
+  isMobile
+}: {
+  page: Page;
+  isMobile: boolean;
+}) => {
+  if (isMobile) {
+    const menuButton = page.getByRole('button', {
+      name: translations.buttons.menu
+    });
+    await expect(menuButton).toBeVisible();
+    await menuButton.click();
+  }
+
+  return page.getByLabel('Search');
+};
+
+test.describe('Search bar optimized', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+  });
+
+  test('should display correct placeholder', async ({ page, isMobile }) => {
+    const searchInput = await getSearchInput({ page, isMobile });
+
+    await expect(searchInput).toBeVisible();
+    await expect(searchInput).toHaveAttribute(
+      'placeholder',
+      'Search 10,700+ tutorials'
```

Comment:
We update this number somewhat frequently- are we able to pull from the translations for this one?